packages/web-components/src/components/rux-classification-marking/rux-classification-marking.tsx

```diff
@@ -97,7 +100,7 @@ export class RuxClassificationMarking {
                             'rux-classification--banner__footer':
                                 isWrapper === true,
                         }}
-                        part="footer-banner"
+                        part="footer-banner footer"
```

Comment:
I thought just footer would be better in the future as a part name, but I'm fine with footer-banner if we want to avoid the extra work of deprecating `footer-banner` as a part

Reply:
can we add "DEPRECATED" in the description for footer-banner?